storage/connect/xindex.cpp

```diff
@@ -1207,7 +1208,7 @@ bool XINDEX::MapInit(PGLOBAL g)
   PCOL    colp;
   PXCOL   prev = NULL, kcp = NULL;
   PDOSDEF defp = (PDOSDEF)Tdbp->To_Def;
-  PDBUSER dup = PlgGetUser(g);
+  PlgGetUser(g);
```

Comment:
better remove the line

Reply:
Done. I treated these functions as they were opaque, because I suspected they were there for a reason. Surely one would optimize the code before committing it beforehand, so there would be a reason for such an unnecessary function call. Even if it was something like "the code breaks if we remove it and we don't know why".